web-portal/backend/prisma/schema.prisma

```diff
@@ -1,17 +1,66 @@
-// This is your Prisma schema file,
-// learn more about it in the docs: https://pris.ly/d/prisma-schema
-
 datasource db {
-  provider = "sqlite"
-  url      = "file:dev.db"
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
 }
 
 generator client {
   provider = "prisma-client-js"
 }
 
 model User {
-  id    String  @id @default(cuid())
-  email String  @unique
-  name  String?
+  id    String @id @unique @default(cuid())
+  org   Org    @relation(fields: [orgId], references: [id])
+  orgId String
+  apps  App[]  @relation("UserApps")
+}
```

Comment:
I think we got off User too soon, since this is the first thing we'll need, we should model how we handle authentication in the database. Do we add `eth_address` here? Or do we link to an authentication interface which handles the implementation?